mysite/static/js/importer.js

```diff
@@ -958,6 +958,21 @@ $(PortfolioEntry.Add.init);
 /*
  *      Re-order projects
  *-------------------------*/
+function getCookie(name) {
```

Comment:
This could have a more specific name, and I feel like we should already have Javascript that does this, but I'm not going to nitpick. It works fine.